src/ntfs/read.rs

```diff
@@ -31,90 +31,221 @@ use transit_model_collection::*;
 
 impl From<Stop> for StopArea {
     fn from(stop: Stop) -> StopArea {
+        if stop.name.is_empty() {
+            warn!("stop_id: {}: for station stop_name is required", stop.id);
```

Comment:
Should we `bail!` instead (and/or possibly panic)? Same question below (at least 3 more times).